views/src/main/res/drawable/bg_outline_light_grey.xml

```diff
@@ -1,14 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-	xmlns:android="http://schemas.android.com/apk/res/android"
-	android:shape="rectangle">
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+	<item>
+		<shape
+			xmlns:android="http://schemas.android.com/apk/res/android"
+			android:shape="rectangle">
 
-	<corners android:radius="1dp" />
-	<stroke android:width="2dp" android:color="@color/light_grey" />
-	<padding android:left="2dp"
-			 android:top="2dp"
-			 android:right="2dp"
-			 android:bottom="2dp" />
-	<solid android:color="@android:color/white" />
+			<corners android:radius="1dp"/>
+			<stroke
+				android:width="2dp"
+				android:color="@color/light_grey"/>
+			<padding
+				android:bottom="2dp"
+				android:left="2dp"
+				android:right="2dp"
+				android:top="2dp"/>
+			<solid android:color="@android:color/white"/>
 
-</shape>
+		</shape>
+	</item>
+
+	<item
+		android:right="18dp">
+		<bitmap
```

Comment:
This approach is REALLY NICE. I thought we were going to have to use a custom view to get the dropdown arrow in